<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationWdgtMain</class>
 <widget class="QWidget" name="ConfigurationWdgtMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Scheduler</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Show new cards for the first time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="new_cards">
            <item>
             <property name="text">
              <string>in the order they were added</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>in random order</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Review memorised cards</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="scheduled_cards">
            <item>
             <property name="text">
              <string>most urgent first</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>in random order</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Hold</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="non_memorised_cards">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>non-memorised cards in your hand</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Autosave after</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="save_after_n_reps">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>repetitions</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="audio_box">
       <property name="title">
        <string>Audio/video</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="media_autoplay">
          <property name="text">
           <string>Start automatically when displaying card</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="media_controls">
          <property name="text">
           <string>Show controls (pause, ...)</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Card presentation</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Show question and answer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="card_presentation">
            <item>
             <property name="text">
              <string>in two boxes with fixed size</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>in two boxes with adaptive size</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>in a single box, top-aligned</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Science</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="upload_science_logs">
          <property name="text">
           <string>Upload anonymous science logs</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Language</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QComboBox" name="languages"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
